contracts/ssnlist.scilla

```diff
@@ -593,7 +623,7 @@ transition UpdateDeleg(ssnaddr : ByStr20, deleg : ByStr20, stake_amt : Uint128,
     is_delete = builtin eq stake_amt uint128_zero;
```

Comment:
should ensure deleg has no buffered deposit

Reply:
@renlulu  what about this?

Reply:
This one is only for remove those delegators that added by mistake? Actually I cannot foresee other use for this during upgrading procedure.